pkg/network/ovn_kubernetes.go

```diff
@@ -260,6 +264,24 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 		var geneve uint32 = uint32(6081)
 		sc.GenevePort = &geneve
 	}
+
+	if sc.PolicyAuditConfig.RateLimit == nil {
+		var ratelimit uint32 = uint32(20)
+		sc.PolicyAuditConfig.RateLimit = &ratelimit
+	}
+	if sc.PolicyAuditConfig.MaxFileSize == nil {
+		var maxfilesize uint32 = uint32(50000000)
+		sc.PolicyAuditConfig.MaxFileSize = &maxfilesize
+	}
+	if sc.PolicyAuditConfig.Destination == "" {
```

Comment:
You're going to want to validate that this is a destination OVN understands; you shouldn't just pass this through directly to the shell script.  (It's also a shell-script unescape exploit).

Reply:
e.g. setting a facility of `$(reboot)`...

Reply:
Yep I added validation on the API side of things, let me know if that's alright https://github.com/openshift/api/pull/853/files#diff-2e98c189fdf696589c6d28124557b9cf96a329e82f790b4e61209351cc68648dR382

Reply:
That's probably fine, then.